### PR TITLE
[gen_l10n] Better blank lines in the header of generated files

### PR DESCRIPTION
**Pull request by AlexV525**
Improve the generated localization files by removing useless blank lines in the header.

localizations_*.dart:
| Before | After |
|--------|-------|
| image | image |

localizations.dart:
| Before | After |
|--------|-------|
| image | image |

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by AlexV525** [comment]
Or maybe we can run the format for the content?

File: packages/flutter_tools/lib/src/localizations/gen_l10n_templates.dart [hunk 1]
```diff
@@ -41,14 +40,14 @@ import 'package:intl/intl.dart' as intl;
...
 /// packages:
 ///
-/// ```
+/// ```yaml
 /// dependencies:
 ///   # Internationalization support.
 ///   flutter_localizations:
 ///     sdk: flutter
-///   intl: any # Use the pinned version from flutter_localizations
+///   intl: any # Pinned by flutter_localizations
```
Comment (christopherfujino) [inline packages/flutter_tools/lib/src/localizations/gen_l10n_templates.dart:48]: Why this change?

Comment (AlexV525) [reply packages/flutter_tools/lib/src/localizations/gen_l10n_templates.dart:48]: Oops, might be a random change that should not submit.

File: packages/flutter_tools/test/general.shard/generate_localizations_test.dart [hunk 2]
```diff
@@ -743,7 +743,8 @@ void main() {
     );
   });
 
-  group('generateLocalizations', () {
+  group('generateLocalizations', ()
```
Comment (christopherfujino) [inline packages/flutter_tools/test/general.shard/generate_localizations_test.dart:746]: Please remove all format-only diffs from this file, per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#formatting

Comment (AlexV525) [reply packages/flutter_tools/test/general.shard/generate_localizations_test.dart]: Ah, raw texts confused me. Removed.

File: packages/flutter_tools/test/general.shard/generate_localizations_test.dart [hunk 3]
```diff
@@ -775,10 +775,12 @@ void main() {
       expect(generator.templateArbFile.path, '/lib/l10n/app_en.arb');
       expect(generator.baseOutputFile.path, '/lib/l10n/bar.dart');
       expect(generator.className, 'Foo');
-      expect(generator.preferredSupportedLocales.single, LocaleInfo.fromString('es'));
+      expect(generator.preferredSupportedLocales.single,
```
Comment (christopherfujino) [inline packages/flutter_tools/test/general.shard/generate_localizations_test.dart:778]: please revert this format change

File: packages/flutter_tools/test/general.shard/generate_localizations_test.dart [hunk 4]
```diff
@@ -775,10 +775,12 @@ void main() {
...
+          LocaleInfo.fromString('es'));
       expect(generator.header, 'HEADER');
       expect(generator.useDeferredLoading, isTrue);
-      expect(generator.inputsAndOutputsListFile?.path, '/gen_l10n_inputs_and_outputs.json');
+      expect(generator.inputsAndOutputsListFile?.path,
```
Comment (christopherfujino) [inline packages/flutter_tools/test/general.shard/generate_localizations_test.dart:782]: please revert this format change

File: packages/flutter_tools/test/general.shard/generate_localizations_test.dart [hunk 5]
```diff
@@ -802,7 +803,8 @@ class FooEn extends Foo {
 ''');
     });
 
-    testWithoutContext('throws exception on missing flutter: generate: true flag', () async {
+    testWithoutContext(
```
Comment (christopherfujino) [inline packages/flutter_tools/test/general.shard/generate_localizations_test.dart:806]: please revert this format change

File: packages/flutter_tools/test/general.shard/generate_localizations_test.dart [hunk 6]
```diff
@@ -826,19 +828,80 @@ flutter:
...
 
       expect(
-        () => generateLocalizations(
-          fileSystem: fs,
-          options: options,
-          logger: BufferLogger.test(),
-          projectDir: fs.currentDirectory,
-          dependenciesDir: fs.currentDirectory,
-        ),
+            () =>
```
Comment (christopherfujino) [inline packages/flutter_tools/test/general.shard/generate_localizations_test.dart:831]: please revert this format change

**Review by christopherfujino — APPROVED** [review]
LGTM

**Review by goderbauer — APPROVED** [review]
LGTM
